yoda_dbt2looker/parser.py

```diff
@@ -250,25 +271,25 @@ def _extract_measures_models(
             exposure_model_views.update(_extract_all_refs(measure.sql))
 
 
-def _extract_calculated_dimensions_models(
+def _extract_models(
     exposure_model_views: set[str],
-    model: dict[str, list[models.Dbt2LookerExploreDimension]],
-    exposure: models.DbtExposure,
+    model: dict[str, list[BaseModel]],
```

Comment:
the name of this argument is a bit unclear - following your decision of the name of the function , consider providing a name that will explain this dictionary